spec/models/user_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { build(:user) }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  context 'validations' do
+    before { create(:user) }
+
+    context 'presence' do
```

Comment:
Please see my comment on `user.rb` about validating `email` and `password` which are two attributes already validated by `devise` for us.